xrootd/client/file_test.go

```diff
@@ -71,3 +72,42 @@ func TestFile_CloseVerify(t *testing.T) {
 		})
 	}
 }
+
+func testFile_ReadAt(t *testing.T, addr string) {
+	client, err := NewClient(context.Background(), addr, "gopher")
+	if err != nil {
+		t.Fatalf("could not create client: %v", err)
+	}
+	defer client.Close()
+
+	fs := client.FS()
+
+	file, err := fs.Open(context.Background(), "/tmp/file1.txt", xrdfs.OpenModeOtherRead, xrdfs.OpenOptionsNone)
```

Comment:
perhaps we should add another (somewhat bigger, let's say 8k) file to further test `ReadAt` ?

Reply:
I agree.

We can do it in `write` test which will write a 8k file to the server and then read it to check back.
Otherwise, we can have a separate test for `read` which will read from the precreated file. 

What do you think?

Reply:
SGTM.